.github/workflows/test.yml

```diff
@@ -9,12 +9,18 @@ on:
 jobs:
   test:
     runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.7"
+          - "3.0"
     steps:
     - uses: actions/checkout@v2
     - name: Set up Ruby
       uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 2.7
+        ruby-version: ${{ matrix.ruby }}
         bundler-cache: true
     - name: Run Rubocop
       run: bundle exec rubocop
```

Comment:
Do we want to run rubocop in a matrix as well? Thoughts on just using https://github.com/theforeman/actions#rubocop & https://github.com/theforeman/actions#gem-test instead?

Reply:
I didn't want to disrupt the rest of the action just yet.
Matrixing Rubocop doesn't hurt. Long term we should move to shared actions, but IMHO don't need today.